### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Feature_Spark Job View_High Contrast: High Contrast Black & White themes are not adapted by the Table Section in 'Spark Job View'

### ISSUE THREAD

**Issue by Manisha370** [microsoft/azure-tools-for-java#3618]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse-NewFeature; #MAS4.3.1; #HighContrast; #C+E-H; #JavatoolingforIntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:**
OS: Windows 10 Enterprise (Version 1903, Build: 18362.356)
MAS Violated: MAS4.3.1

**Note:**
**Similar issue is observed for "Application Graph" page**

**Repro Steps:**
1.Turn on High Contrast Black & White 
2.Open Java EE - Eclipse application in light theme in light theme
3.Install 'Azure Toolkit for Eclipse' from Marketplace
4.Open Eclipse and install  the Scala plug-in 
5.Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
6.In Azure Explorer section, tab till HDInsight and expand it. Select 'Spark on HDInsight (Scala)'. Tab till Next and press enter
7.Tab till Next button and press enter. Press Finish button
8.In Project Explorer, Expand the created Scala project right-click src, New-->Other and press enter
9.Tab till Scala Wizards, expand and select 'Scala Object' and press next
10.Enter the name of the file and press Finish button
11.Open Scala file and write any code in the editor
12.In Project Explorer view, right-click your project, choose Azure-->Submit Spark Application to HDInsight
13.Enter all the required details and press submit button
14.In Azure Explorer tab, tab till HDInsight-->Cluster-->Job and press enter
15.Verify whether the 'Table section' is adapting the high contrast black & white themes or not

**Actual Result:**
The 'table section' in "Spark Job View" is not adapting the high contrast black & white themes 

**Expected Result:**
The 'table section' in "Spark Job View" should adapt the high contrast black & white themes

**User Impact:**
Users with color blindness will find difficulty in viewing the table, if it doesn't adapt the high contrast black & white themes

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}

![Bug7_Table section and Application graph sections are not adapting high contrast black themes](https://user-images.githubusercontent.com/51435937/65509828-11491e80-def1-11e9-8f44-b143a0f2bb1a.png)
![Bug7_Table section and Application graph sections are not adapting high contrast white](https://user-images.githubusercontent.com/51435937/65509830-11491e80-def1-11e9-9590-3d3b75761891.png)

**Comment by t-rufang** [comment]
The job view page is wrote as static web pages with the help of JavaFX WebView which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749718
